core/src/test/java/org/infinispan/security/CacheAuthorizationTest.java

```diff
@@ -91,6 +95,49 @@ public Void run() {
       });
    }
 
+   public void testAdminCombinations() throws Exception {
+      List<Supplier<Void>> calls = Arrays.asList(() -> {
```

Comment:
You never use the return type, so we can make this `List<Runnable>` and then item can be a one liner e.g: 
`() -> cacheManager.getGlobalComponentRegistry()`.